packages/core/src/components/fields/ArrayField.js

```diff
@@ -570,7 +571,7 @@ class ArrayField extends Component {
         disabled={disabled}
         readonly={readonly}
         required={required}
-        label={label}
+        label={title}
```

Comment:
Will this give us the same value that `label` had earlier for non-multiselect ArrayFields?

Reply:
For all other widgets in `ArrayField.js`, `label` is not set and thus undefined. Other fields set it similarly to title ([StringField](https://github.com/Parnswir/react-jsonschema-form/blob/fix/arrayfield_label/packages/core/src/components/fields/StringField.js#L47), [NumberField](https://github.com/Parnswir/react-jsonschema-form/blob/fix/arrayfield_label/packages/core/src/components/fields/BooleanField.js#L61)).

Again, I cannot tell if this is the way it should behave, but for use-cases like i18n (if done client-side), one will need the name or title of the schema property in custom widgets.

Reply:
> Again, I cannot tell if this is the way it should behave, but for use-cases like i18n (if done client-side), one will need the name or title of the schema property in custom widgets.

Sorry, I don't think I understood what you meant here. Could you explain further?

Reply:
I was just trying to motivate my use-case again 😄

Reply:
I'm still a bit confused, do you mind explaining that comment again? "Again, I cannot tell if this is the way it should behave, but for use-cases like i18n (if done client-side), one will need the name or title of the schema property in custom widgets."

Reply:
ok, sorry for the misunderstanding. All I mean is I don't know how exactly the `label` prop is defined in general.
For my use-case, basically a dynamic translation of select options in the frontend, all I need is the value of the property name (or title), so that I know which translation key to choose. Of course it's not that simple and I can't show code, but that's the point where the missing name/title/label value in custom multiselects is a problem, hence the issue/PR 😄

Reply:
@epicfaace What can I do to get this merged?

Reply:
Sorry for being pushy, but we're working around this problem for three months now and this PR would help a lot. Can I do anything to help get this merged?